servlet/src/test/java/org/togglz/servlet/test/util/HttpServletRequestHolderTest.java

```diff
@@ -24,9 +25,12 @@
     public static WebArchive createDeployment() {
         return Deployments.getBasicWebArchive()
             .addClass(HttpServletRequestHolderServlet.class)
+            .addClass(TogglzFilterServlet.class)
             // we don't need to bootstrap Togglz here as we only test the request holder
             .setWebXML(Packaging.webAppDescriptor()
                 .contextParam("org.togglz.FEATURE_MANAGER_PROVIDED", "true")
+                .filter(TogglzFilter.class, "/*")
```

Comment:
Could you add a comment here to explain why the filter is added twice in the test! Just to make sure that nobody removes the duplicated registration by accident in a few years. :laughing: